### PR TITLE
Feat: add final rhn.conf

### PR DESCRIPTION
**Pull request by mbologna**
# Description

Adds the resulting rhn.conf after enabling SSO.

# Target branches

Which documentation version does this PR apply to?

- [X] Master (Default)
- [X] Manager-4.3
- [X] Manager-4.2

### CODE REVIEW

File: modules/administration/pages/auth-methods-sso-example.adoc [hunk 1]
```diff
@@ -40,6 +40,76 @@ java.sso.onelogin.saml2.idp.single_logout_service.url
...
+#
+# If 'strict' is True, then the Java Toolkit will reject unsigned
+# or unencrypted messages if it expects them signed or encrypted
+# Also will reject the messages if not strictly follow the SAML
+#
+# WARNING: In production, this parameter setting parameter MUST be set as "true".
+# Otherwise your environment is not secure and will be exposed to attacks.
+# Enable debug mode (to print errors)
+# Identifier of the SP entity (must be a URI)
+java.sso.onelogin.saml2.sp.entityid = https://suma-long-42-srv.mgr.suse.de/rhn/manager/sso/metadata
```
Comment (jcayouette) [inline modules/administration/pages/auth-methods-sso-example.adoc:71]: Should we be using our reference servers in our examples? 

Per the SUSE style guide: Use http://www.example.com/ and http://www.example.org/ as example domains. Both domains were registered for use in documentation.

Comment (jcayouette) [reply modules/administration/pages/auth-methods-sso-example.adoc:71]: It is admittedly a bit awkward with the identity server. We should probably come up with our own terms for domains and servers names. From a user perspective though, the usecase may not be very clear based on our infra.

File: modules/administration/pages/auth-methods-sso-example.adoc [hunk 2]
```diff
@@ -40,6 +40,76 @@ java.sso.onelogin.saml2.idp.single_logout_service.url
...
+# Public x509 certificate of the IdP
+java.sso.onelogin.saml2.idp.x509cert = -----BEGIN CERTIFICATE----- MIIClzCCAX8CBgGC+tPbVjANBgkqhkiG9w0BAQsFADAPMQ0wCwYDVQQDDARTVU1BMB4XDTIyMDkwMTIwNTEwNFoXDTMyMDkwMTIwNTI0NFowDzENMAsG
+A1UEAwwEU1VNQTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAMNSWJAalB5mShTkMBO5mrsOosyheEL8/A37WvuqDPwwEfm4x0cG7gmMHvONxYXZk+LRyzoQl2sBrNFrbMuwu5dnah5ZSMxQyUu697S280m4vIiegGaFdbgH+g4FGBu
+eSis1ssMzTcES+NUuI7pLkMLNmSQtncESnoL9q2SyeQSwYtr5dz1ydl6IzjwtaWeyQ9EGJNtJtLk3U4+arLPCpHAwqFAnLO9NeYcRDNUKhNBs1v5mHP+L066PZu1/DkE0mSgy/+qXaS0CgZVKqz8qB+bvHVuAq9W60g1CjqZKbwvPu72p/7+d8z
+9DxXPIZ1uxdqn19q/kLEP2TYLtgQobSHECAwEAATANBgkqhkiG9w0BAQsFAAOCAQEAga+raLMJDo/P/yN1Z6SGGocK227WFqovBiE/mLYlp5Ff0+0jS1US1plSppJ94xOr8j0m7HW0Wu5xCz6oOhzXTEtnfIbeRyr1Rms3BWdxyXgQ9bWUeZMWZ
+HfDkTbhgRRmjDEwSSfEXRKQNvw41CpnlB36I0++ejgGnjDvH7BbkCaoW55JF5j6DT/WYR0n7MkEl2Ova9CH0e9X7Gny8iOAg26oziy06uy3P/lx9Z9RmHnvpvN/Q34SGEq9z/HlQVuP12UPj//iT21Jc17OOZFsZQXlGFTG6bXKmO42W8FdUDJU
+ONoXZgjMb3eC7U691YyeowoqTY7mJKxNPprYY/lL0w== -----END CERTIFICATE-----
+
+# Organization
+java.sso.onelogin.saml2.organization.name = Michele Bologna
```
Comment (jcayouette) [inline modules/administration/pages/auth-methods-sso-example.adoc:102]: For example users, use free-software mascots, such as Tux (Linux Kernel), Wilber (The GIMP), Geeko (SUSE), Foxkeh (Firefox), Konqi (KDE), or Duke (Java).

I would stay away from using personal names in  our examples.

File: modules/administration/pages/auth-methods-sso-example.adoc [hunk 3]
```diff
@@ -40,6 +40,76 @@ java.sso.onelogin.saml2.idp.single_logout_service.url
...
+java.sso.onelogin.saml2.organization.displayname = Michele Bologna
+java.sso.onelogin.saml2.organization.url = https://suma-long-42-srv.mgr.suse.de
+java.sso.onelogin.saml2.organization.lang = 
+
+# Contacts
+java.sso.onelogin.saml2.contacts.technical.given_name = Michele Bologna
+java.sso.onelogin.saml2.contacts.technical.email_address = mb@suse.com
```
Comment (jcayouette) [inline modules/administration/pages/auth-methods-sso-example.adoc:109]: I am not sure you want to use your personal email here. :)

File: modules/administration/pages/auth-methods-sso-example.adoc [hunk 4]
```diff
@@ -40,6 +40,76 @@ java.sso.onelogin.saml2.idp.single_logout_service.url
 java.sso.onelogin.saml2.idp.x509cert
 ----
 
+Here is an example of the final SUSE Manage's rhn.conf:
```
Comment (keichwa) [inline modules/administration/pages/auth-methods-sso-example.adoc:43]:
```suggestion
Here is an example of [path]``rhn.conf`` on {productname} after enabling SSO:
```
It's a proposal based on the description of this PR.

Comment (jcayouette) [reply modules/administration/pages/auth-methods-sso-example.adoc:43]: Looks good to me. Thanks Karl. 👍

**Comment by mbologna** [comment]
Thanks for the precious indications on style and personal addresses - I force pushed to remove the history ;-)

**Review by keichwa — APPROVED** [review]

**Pull request merged**